rotkehlchen/chain/ethereum/oracles/uniswap.py

```diff
@@ -443,5 +454,14 @@ def get_pool_price(
         if ZERO in (reserve_0, reserve_1):
             raise DefiPoolError(f'Uniswap pool for {token_0}/{token_1} has asset with no reserves')
 
+        # Ignore pools with too low single side-liquidity. Imperfect approach to avoid spam
+        # pylint: disable=unexpected-keyword-arg  # no idea why pylint sees this here
+        price_0 = Inquirer().find_usd_price(token_0, skip_onchain=True)
+        price_1 = Inquirer().find_usd_price(token_1, skip_onchain=True)
```

Comment:
I believe this breaks a little bit the sense with the oracles since is something executed for all the tokens. You are querying uniswap and at the same time other oracles. Is good if you only want to use on chain prices but here you could also return directly the price if available

Reply:
I don't understand. If you don't skip on-chain this becomes an infinite loop.